fastai/transforms.py

```diff
@@ -88,7 +88,7 @@ def googlenet_resize(im, targ, min_area_frac, min_aspect_ratio, max_aspect_ratio
     return out
 
 def cutout(im, n_holes, length):
-    *_,h,w = im.shape
+    r,c,_* = im.shape
```

Comment:
this function does not reference r,c variable instead of h,w. why do we need r,c?